contrib/babelfishpg_tsql/src/catalog.h

```diff
@@ -176,6 +176,7 @@ typedef FormData_authid_user_ext *Form_authid_user_ext;
 #define BBF_VIEW_DEF_NUM_COLS 8
 #define BBF_VIEW_DEF_FLAG_IS_ANSI_NULLS_ON (1<<0)
 #define BBF_VIEW_DEF_FLAG_USES_QUOTED_IDENTIFIER (1<<1)
+#define BBF_VIEW_DEF_FLAG_CREATED_IN_OR_AFTER_2_4 (0<<2)
```

Comment:
we should not use version number in macro, a comment can mention version number. Macro could be something like _CREATED_WITH_ORIGINAL_SQL_DEF

Reply:
The MACRO with the same name is merged in 2_X_DEV. Let's keep it same in 3_X